### ISSUE TITLE
Error "The cloud file provider is not running"

### ISSUE THREAD

**Issue by Skoaz** [ShareX/ShareX#3723]
Hello guys,

I've recently installed ShareX, and everytime the application starts running or when I try to open it, this error appears : 
![image](https://user-images.githubusercontent.com/44904695/48276264-e45e5b00-e447-11e8-94d6-48551bd24734.png)

I've tried to uninstall it and reinstate, but it didn't work. So I was wondering if some of you could help me,

Thank you :)

**Comment by artfabrique** [comment]
does your username has some language-specific characters? It seems it can not read the config file from C:\Users\**%USERNAME%**\Documents\ShareX

**Comment by Skoaz** [comment]
Hello @artfabrique,

The name of my PC was DESKSTOP- with a serie of numbers. I renamed it in my real first name, but there's no document of Share in C:\Users\glefe\Documents : 

![image](https://user-images.githubusercontent.com/44904695/48300428-79f7fa00-e4dd-11e8-8608-5fef0fa79b73.png)

Thank you

**Comment by artfabrique** [comment]
@Skoaz Hm... do you use portable version?

**Comment by Skoaz** [comment]
@artfabrique  I downloaded this one ; 
![image](https://user-images.githubusercontent.com/44904695/48316221-13f49b00-e5e1-11e8-9dec-9da7eb00499f.png)

The version that the site proposes, I don't think it's the portable version.

**Comment by guillermo** [comment]
Same here.

**Comment by guillermo** [comment]
Fixed and found the issue.

To reproduce:

1. Ensure that one have installed one drive, and the documents folder is sync with one drive.
2. Next, ensure that the ShareX files are saved in one drive, and that no local copy exists.
3. Stop one drive or log out one drive.
4. Launch ShareX.

I was able to get the same exception from the command line:

![image](https://user-images.githubusercontent.com/7533/78160561-6fc4fb80-7444-11ea-8d93-559c34026633.png)

And this is how this case is being display in the windows ``explorer.exe``
![image](https://user-images.githubusercontent.com/7533/78160636-8cf9ca00-7444-11ea-9400-1e282222ae13.png)

I reach this state by not using ShareX in over a year, and coming back to use it in a new computer, where without knowing, my config files where in the new computer (Now that is working, i have to say is quite cool that I don't need to reconfigure anything :-P ).

It might also happen with Dropbox or Google Drive.

It might make sense to inform the user about the meaning of this exception somehow.
